src/account/operations/transaction/prepare_output.rs

```diff
@@ -65,6 +65,16 @@ impl AccountHandle {
                 first_output_builder = first_output_builder
                     .add_feature(Feature::Metadata(MetadataFeature::new(metadata.as_bytes().to_vec())?));
             }
+
+            if let Some(issuer) = features.issuer {
+                first_output_builder = first_output_builder
+                    .add_feature(Feature::Issuer(IssuerFeature::new(Address::try_from_bech32(&issuer)?.1)));
+            }
```

Comment:
Do we really want to consider the issuer here? If someone puts an issuer without a `nft_id`, it would end up in an error as a BasicOutput can't have an issuer.

Reply:
I'm fine actually erroring so that the user is aware of the mistake but maybe it at least deserves a small comment to explain

Reply:
I removed the line, since this code is not reached. If there is an nft_id, it will use the prepare_nft_output function with the cloned options, so this way no error is thrown.

Reply:
Yeah but what I'm saying is: what happens if you put an issuer but no NFT ID ?

Reply:
Maybe it should just throw an error on l.70 then?